lms/static/js/instructor_dashboard/membership.js

```diff
@@ -454,21 +456,40 @@ such that the value can be defined later than this assignment (file load order).
                 $taskResSection.append($idsList);
                 for (j = 0, len1 = ids.length; j < len1; j++) {
                     identifier = ids[j];
-                    $idsList.append($('<li/>', {
-                        text: identifier
-                    }));
+                    if (identifier === message) {
+                        if (!$idsList.has( 'li' ).length) {
+                            $taskResSection.find('h3').text(message);
+                        } else {
+                            $idsList.append($('<h3/>', {
+                                text: identifier
+                            }));
+                        }
+                    } else {
+                        $idsList.append($('<li/>', {
+                            text: identifier
+                        }));
+                    }
                 }
                 return displayResponse.$task_response.append($taskResSection);
             };
             if (successes.length && dataFromServer.action === 'add') {
                 // Translators: A list of users appears after this sentence;
                 renderList(gettext('These users were successfully added as beta testers:'), (function() {
```

Comment:
Yeah, firstly I consider it to separate the block `These users were successfully added as beta testers:`
But `renderList` method only render one list i.e either the `Active` or `Inactive`.
So that's why I have added the functionality of `Inactive users` under this part.